.pr_agent.toml

```diff
@@ -0,0 +1,39 @@
+[config]
+model="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+model_turbo="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+fallback_models=["gpt-4"] # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+
+[github_action_config]
+auto_review = true
+auto_describe = true
+auto_improve = true
+pr_actions = ["opened", "reopened", "ready_for_review", "review_requested"]
+
+[pr_reviewer] # /review #
+# enable/disable features
+require_score_review=false
+require_tests_review=true
+require_estimate_effort_to_review=true
+require_can_be_split_review=false
+require_security_review=true
+require_ticket_analysis_review=true
+# general options
+num_code_suggestions=0
+inline_code_comments = true
+ask_and_reflect=false
```

Comment:
I'm curious to know what `ask_and_reflect` does

Reply:
Looks like it'll rate its own code suggestions 
<img width="770" alt="Screenshot 2024-10-14 at 1 40 40 PM" src="https://github.com/user-attachments/assets/307bf789-65b1-407a-9ad2-770661275d0e">

Reply:
I think it'd be worth adding this option if there's no drawback to set that on